.github/workflows/CI.yml

```diff
@@ -59,11 +59,20 @@ jobs:
         --slave /usr/bin/gfortran gfortran /usr/bin/gfortran-${GCC_V} \
         --slave /usr/bin/gcov gcov /usr/bin/gcov-${GCC_V}
 
+    - name: Install GFortran Windows
+      if: contains(matrix.os, 'windows')
+      run: |
+        Invoke-WebRequest -Uri $Env:GCC_DOWNLOAD -OutFile mingw-w64.zip
+        Expand-Archive mingw-w64.zip
+        echo "$pwd\mingw-w64\mingw64\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+      env:
+        GCC_DOWNLOAD: "https://github.com/brechtsanders/winlibs_mingw/releases/download/9.4.0-9.0.0-msvcrt-r2/winlibs-x86_64-posix-seh-gcc-9.4.0-mingw-w64-9.0.0-r2.zip"
```

Comment:
I guess it was about time we get rid of the GCC 8.1 version on Windows. This can close https://github.com/fortran-lang/fpm/issues/537?

Reply:
Yes. Though I had a problem that the gcc packaged by choco had some incomplete or old headers needed for isatty on Windows, but winlibs works perfectly fine.